### PR TITLE
Bug 1622063 - Add UA override for wp1-ext.usps.gov

### PR DESCRIPTION
**Pull request by miketaylr**
r? @ksy36

### CODE REVIEW

**Comment by miketaylr** [comment]
https://github.com/webcompat/web-bugs/issues/29867

File: src/data/ua_overrides.js [hunk 1]
```diff
@@ -607,6 +607,25 @@ const AVAILABLE_UA_OVERRIDES = [
       },
     },
   },
+  {
+    /*
+     * Bug 1622063 - UA overrides for bracketchallenge.ncaa.com
+     * Webcompat issue #49886 - https://webcompat.com/issues/49886
```
Comment (ksy36) [inline src/data/ua_overrides.js]: should be wp1-ext.usps.gov? and webcompat issue #29867

Comment (miketaylr) [reply src/data/ua_overrides.js]: argh, rookie move. thanks :)

**Pull request merged**